sentry_config.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import os
+import sentry_sdk
+from sentry_sdk.integrations.httpx import HttpxIntegration
+from sentry_sdk.integrations.logging import LoggingIntegration
+
+
+def init_sentry():
+    """Initialize Sentry SDK with proper configuration."""
+    sentry_sdk.init(
```

Comment:
This should be from `SENTRY_DSN` env. And if the env is not available, just skip it (but show a warning).